src/components/com_tjdashboard/administrator/views/dashboard/tmpl/edit.php

```diff
@@ -31,17 +31,17 @@
 $layout  = $isModal ? 'modal' : 'edit';
 $tmpl    = $isModal || $input->get('tmpl', '', 'cmd') === 'component' ? '&tmpl=component' : '';
 
-JFactory::getDocument()->addScriptDeclaration('
-	Joomla.submitbutton = function(task)
-	{
-		if (task == "dashboard.cancel" || document.formvalidator.isValid(document.getElementById("dashboard-form")))
-		{
-			jQuery("#permissions-sliders select").attr("disabled", "disabled");
-			' . $this->form->getField('description')->save() . '
-			Joomla.submitform(task, document.getElementById("dashboard-form"));
-		}
-	};
-');
+// JFactory::getDocument()->addScriptDeclaration('
+// 	Joomla.submitbutton = function(task)
+// 	{
+// 		if (task == "dashboard.cancel" || document.formvalidator.isValid(document.getElementById("dashboard-form")))
+// 		{
+// 			jQuery("#permissions-sliders select").attr("disabled", "disabled");
+// 			' . $this->form->getField('description')->save() . '
+// 			Joomla.submitform(task, document.getElementById("dashboard-form"));
+// 		}
+// 	};
+// ');
```

Comment:
is there any alternative to this code? If not then what happens to this functionality @shindebalu